src/cli/EmbeddedScripting.i

```diff
@@ -9,6 +9,8 @@
 //%import <utilities/core/CommonImport.i>
 //%import <utilities/Utilities.i>
 
+%ignore embedded_files::fileNames;
```

Comment:
```suggestion
%ignore embedded_files::fileNames;
%ignore embedded_files::extractFile;
// Was formerly used but isn't anymore
%ignore embedded_files::findFirstFileByName;
```

Reply:
Also, I do not understand one bit why we `#define UTILITIES_API` above. And neither why this file does not have header guards?

Reply:
So, I left the methods in there since they don't hurt and could be used for debugging something. But I reordered the file and added header guards